components/ai_chat/resources/page/components/model_selector/style.module.scss

```
@@ -0,0 +1,142 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+.anchorButton {
+  --leo-button-color: var(--leo-color-text-tertiary);
+}
+
+.anchorButtonContent {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  gap: var(--leo-spacing-s);
+  overflow: hidden;
+  max-width: 100%;
+}
+
+.anchorButtonText {
+  font: var(--leo-font-small-regular);
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.anchorButtonOpen {
+  --leo-button-color: var(--leo-color-icon-interactive);
+}
+
+.anchorButtonIcon {
+  --leo-icon-size: 18px;
+  transform: rotate(0deg);
+  transition: transform 0.2s ease;
+}
+
+.anchorButtonIconOpen {
+  transform: rotate(180deg);
+  transition: transform 0.2s ease;
+}
+
+.buttonMenu {
+  leo-menu-item {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: var(--leo-spacing-l);
+    padding: var(--leo-spacing-s) var(--leo-spacing-m);
+    max-width: 270px;
+    font: var(--leo-font-default-regular);
+    --leo-icon-size: 20px;
+    --leo-icon-color: var(--leo-color-icon-default);
```

nit: css variables at top of block


```suggestion
    --leo-icon-size: 20px;
    --leo-icon-color: var(--leo-color-icon-default);

    display: flex;
    align-items: center;
    justify-content: space-between;
    gap: var(--leo-spacing-l);
    padding: var(--leo-spacing-s) var(--leo-spacing-m);
    max-width: 270px;
    font: var(--leo-font-default-regular);
```